### ISSUE TITLE
viewHelp is broken

### ISSUE THREAD

**Issue by DanGrayson** [Macaulay2/M2#1761]
If I install a package called "Foo", say, to my user application directory, and then load it, then `viewHelp Foo` shows the documentation page for `null` instead of the documentation page for the package.  This worked before.

**Comment by mahrud** [comment]
I can't replicate this, are you sure the issue is not local?
```m2
root@53cdaaa7fe41:/home/macaulay# WWWBROWSER=echo M2
Macaulay2, version 1.17.0.1
with packages: ConwayPolynomials, Elimination, IntegralClosure, InverseSystems, LLLBases, MinimalPrimes, PrimaryDecomposition,
               ReesAlgebra, Saturation, TangentCone

i1 : installPackage "Foo"
 -- warning: found 1 documentation node(s) not listed as a subnode

o1 = Foo

o1 : Package

i2 : viewHelp Foo
/root/.Macaulay2/local/share/doc/Macaulay2/Foo/html/index.html

i3 : get "/root/.Macaulay2/local/share/doc/Macaulay2/Foo/html/index.html"

o3 = <?xml version="1.0" encoding="utf-8" ?>
...
```

**Comment by mahrud** [comment]
Here is the content of `Foo.m2` in the same directory:
```m2
newPackage("Foo", Headline => "hoohoho")

export {"aa"}

aa = 12

beginDocumentation()

document {Key => "Foo", Headline => "Fooo"}
document {Key => "aa", Headline => "aaaa"}
```

**Comment by mahrud** [comment]
By the way, I tested this in a fresh docker container built from the nightly package produced by github actions, so I'm pretty sure it's not just working for me. See this [manual](https://github.com/mahrud/M2/blob/quickfix/actions/M2/BUILD/docker/nightly/README.md).

**Comment by mahrud** [comment]
I added a line to throw an error instead of open the help for "null" if a page doesn't exist in #1758.
But the issue about new packages doesn't seem to be reproducible.

**Comment by DanGrayson** [comment]
I think you'll be able to reproduce it if you try it with this as the contents of Foo.m2 : 
```m2
newPackage ( "Foo", Headline => "foo" )
```

**Comment by mahrud** [comment]
Ah, then I agree with this statement of yours:

_Originally posted by @DanGrayson in https://github.com/Macaulay2/M2/issues/306#issuecomment-110968167_:
> Maybe `installPackage` should give an error in that case.

**Comment by DanGrayson** [comment]
> Ah, then I agree with this statement of yours:
> 
> _Originally posted by @DanGrayson in [#306 (comment)](https://github.com/Macaulay2/M2/issues/306#issuecomment-110968167)_:
> 
> > Maybe `installPackage` should give an error in that case.

That would be a fine way to fix it.

**Comment by mahrud** [comment]
It's unfortunate that so many packages don't have top level docs. This should be fixed in #1758 for now.
